tracer/src/Datadog.Trace/IAST/Range.cs

```diff
@@ -6,13 +6,14 @@
 #nullable enable
 
 using System;
+using System.Collections.Generic;
 using System.Diagnostics.CodeAnalysis;
 
 namespace Datadog.Trace.Iast;
 
 internal readonly struct Range : IComparable<Range>
 {
-    public Range(int start, int length, Source source)
+    public Range(int start, int length, Source? source)
```

Comment:
Why source can be nullable in range?

Reply:
Because of the sensitive ranges, which have no source, and I found more convenient to reuse the range struct instead of creating a base class